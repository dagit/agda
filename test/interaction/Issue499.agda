
module Issue499 where

data A : Set where
  a : A

data B : Set where
  b : .A → B

data C : B → Set where
  c : C (b a)

f : ∀ {i} → C i → A
f x = {!!} -- Hitting C-c C-c x
