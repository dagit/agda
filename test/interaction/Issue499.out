Prelude> Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue499 (Issue499.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue499.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : A\n" nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> ((last . 2) . (agda2-make-case-action '("f c = ?")))
((last . 1) . (agda2-goals-action '(0)))
Agda2> 